### PR TITLE
add nft marketplaces coresky

### PR DESCRIPTION
**Pull request by csgithub007**
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Please enable "Allow edits by maintainers" while putting up the PR.
3. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Name (to be shown on DefiLlama):
Coresky

##### Twitter Link:
https://twitter.com/Coreskyofficial

##### Website Link:
https://www.coresky.com/

##### Github org/user (Optional, if your code is open source, we can track activity):

### CODE REVIEW

**Comment by llamatester** [comment]
Error while running adapter at projects/coresky: 
        
 
  
    
    
    Error: Multicall failed!
    Truncated error stack:
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/abi/index.js:286:35

File: projects/coresky/index.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const { staking } = require("../helper/staking");
+const { pool2 } = require("../helper/pool2");
+
+const coresky_ETHEREUM = "0x179886f229f9cd52016c6dbf4be66e18111ec6ab";
```
Comment (g1nt0ki) [inline projects/coresky/index.js:4]: this does not look like an erc20 token address, also dont see any tokens staked in the controller contract

**Review by g1nt0ki — CHANGES_REQUESTED** [review]
.

**Comment by llamatester** [comment]
Error while running adapter at projects/coresky: 
        
 
  
    
    
    Error: Multicall failed!
    Truncated error stack:
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/abi/index.js:286:35

**Comment by jason0513** [comment]
Yeah, This is a market contract here , and I would like to display our market in the NFT market position. 
What should I do？

**Comment by jason0513** [comment]
Yes, our platform is like the trading platforms of Opensea and blur, and we charge a certain transaction commission. I want to place it in the NFT marketplaces for ranking.
Look at the picture below
market

**Comment by slasher125** [comment]
> Yes, our platform is like the trading platforms of Opensea and blur, and we charge a certain transaction commission. I want to place it in the NFT marketplaces for ranking. Look at the picture below market

henlo. can u ping me on discord, i'll assist with adding to nft-server

**Comment by jason0513** [comment]
Okay, I join in your Discord.

**Pull request closed without merging**